### DISCUSSION TITLE
Function to calculate the Lunation Number

### DISCUSSION THREAD

**Discussion by meszarosrob** [jasonsturges/lunarphase-js#1] [Ideas]
👋 What do you think of having out-of-the-box function for calculating the [Lunation Number](https://en.wikipedia.org/wiki/New_moon#Lunation_Number)? 🤔

Here's a way to do it:

```js
// getJulianDate(new Date('1923-01-17 02:41'))
const BASE_LUNATION_JULIAN_DATE = 2423436.6115277777

const lunarNumber = (date = new Date()) => {
    return Math.round((getJulianDate(date) - BASE_LUNATION_JULIAN_DATE) / LUNAR_MONTH) + 1
}
```

**Comment by jasonsturges** [comment]
Interesting - yeah, that'd be cool.  Thanks, I'll get that in.

**Comment by jasonsturges** [comment]
@meszarosrob Going to be working on this package this week, and hope to incorporate this soon.

Thanks, again.

**Comment by jasonsturges** [comment]
@meszarosrob Got it in - per your recommendation of Brown's Lunation Number.

Brown Lunation Number (BLN), per Wikipedia:

> ...defines lunation 1 as beginning at the first new moon of 1923, the year when Ernest William Brown's lunar theory was introduced in the American Ephemeris and Nautical Almanac.[citation needed] Lunation 1 occurred at approximately 02:41 UTC, January 17, 1923.

Still refactoring this project around a bit, but I like this addition.

Available in v1.0.6

**Reply by meszarosrob** [reply]
@jasonsturges yey! So cool! Thanks 🙏
